src/lib/utils/customerUtils.ts

```diff
@@ -57,3 +57,21 @@ export const getTariffPlan = (customer: CustomerRecord, currentSite: SiteRecord)
   const tariffPlanId = customer.tariffPlanId;
   return currentSite.tariffPlans.find((plan:TariffPlanRecord) => plan.id === tariffPlanId);
 }
+
+export const EMPTY_CUSTOMER : CustomerRecord = {
+  id: '',
+  name: '',
+  meterNumber: 0,
+  tariffPlanId: '',
+  isactive: false,
+  hasmeter: false,
+  outstandingBalance: '',
```

Comment:
should be 0 instead of empty string. Otherwise the Total Owed doesn't show 0 in the customer main like it does with other customers. 

![image](https://user-images.githubusercontent.com/35501399/110226393-933dc500-7ea3-11eb-90c1-b45187dd4ec1.png)